mkdocs.yml.in

```diff
@@ -52,10 +52,10 @@ nav:
     - setup.md
     - example.md
     - contributing.md
-  - Generator: '!include ./djinni-generator/mkdocs.yml'
-  - Support Lib: '!include ./djinni-support-lib/mkdocs.yml'
+  - Generator <code>${VERSION_djinni_generator}</code>: '!include ./djinni-generator/mkdocs.yml'
+  - Support Lib <code>${VERSION_djinni_support_lib}</code>: '!include ./djinni-support-lib/mkdocs.yml'
```

Comment:
The `VERSION_...` variables contain the version of each of the submodules.

the `code` tags will be rendered into the resulting html and can then be styled to look like small version badges.